genisys/modules/kernelparameter.py

```diff
@@ -0,0 +1,45 @@
+from typing import Self
+from pathlib import Path
+
+from . import base
```

Comment:
Change this to `from genisys.modules import base`. It will be more consistent when installed globally.

Reply:
Done.